YAXLib/Custom/Locker.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors.
```

Comment:
This class is responsible for blocking recursive calls to `ICustomSerializer` orn `IKnownType` instances. This way the `YAXSerializer` can de/serialize a "raw version", which can then be modified.